crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -1536,6 +1537,17 @@ def execute_scripts_and_wait_for_finish(nodes, script, seconds)
         end
       end
 
+      # remove crowbar-ui package lock for future upgrades
+      def unlock_crowbar_ui_package
+        ui_unlock = run_cmd("sudo zypper-retry removelock 'crowbar-ui*'")
+        unless ui_unlock[:exit_code].zero?
+          raise_node_upgrade_error(
+            "Removing crowbar-ui package lock has failed. " \
+            "Make sure to remove it before next upgrade."
```

Comment:
I guess that if this fails it would require some manual action to fix it as a number of different things could prevent this from working. Maybe repeating + advice to manually check if the error persist is the way to go

Reply:
Message updated. Not sure if the wording makes sense...